site/domain-resource.md

```diff
@@ -0,0 +1,35 @@
+
+# Domain Resource
+
+In this guide, we outline how to set up and configure your own domain resource which can be used to configure your WLS domain. The domain resource which can then be used to start the Kubernetes artifacts of the corresponding domain. 
```

Comment:
The domain resource which can then be used -> Then, you can use the domain resource to start...